fp87-sim.c build fails

```
gcc -c -imacros emu_config.h -MD -DCFLAGS_STR=" -fplan9-extensions -Wstrict-aliasing=3 -Wall -Wstrict-prototypes -Wmissing-declarations -Wnested-externs -fms-extensions -pthread -Wno-unused-result -Wcast-qual -Wwrite-strings -Wundef -Wno-address-of-packed-member -ggdb3 -fpie -fpic -O2 -flto=auto    " -iquote ../../../../src/include -iquote ../../../../src/plugin -iquote /home/[user]/proj/newnewdosemu/20250906/src/base/bios/x86 -iquote /home/[user]/proj/newnewdosemu/20250906/src/include -iquote /home/[user]/proj/newnewdosemu/20250906/src/base/lib -iquote ../../../..  -I/home/[user]/proj/newnewdosemu/20250906/src/emu-i386/simx86 -Dlinux -DDOSEMU -fplan9-extensions -Wstrict-aliasing=3 -Wall -Wstrict-prototypes -Wmissing-declarations -Wnested-externs -fms-extensions -pthread -Wno-unused-result -Wcast-qual -Wwrite-strings -Wundef -Wno-address-of-packed-member -ggdb3 -fpie -fpic -O2 -flto=auto    -o fp87-sim.o fp87-sim.c
fp87-sim.c: In function ‘Fp87_op_sim’:
fp87-sim.c:1183:1: error: label at end of compound statement
 1183 | fp_ok:
      | ^~~~~
make[2]: *** [/home/[user]/proj/newnewdosemu/20250906/src/Makefile.common.post:83: fp87-sim.o] Error 1
make[2]: Leaving directory '/home/[user]/proj/newnewdosemu/20250906/src/base/emu-i386/simx86'
make[1]: *** [/home/[user]/proj/newnewdosemu/20250906/src/arch/linux/Makefile.main:131: base/emu-i386/simx86] Error 2
make[1]: Leaving directory '/home/[user]/proj/newnewdosemu/20250906/src'
make: *** [Makefile:42: default] Error 2
```

Just pulled 5min ago.

--- a/src/base/emu-i386/simx86/fp87-sim.c
+++ b/src/base/emu-i386/simx86/fp87-sim.c
@@ -804,7 +804,7 @@ fcom00:			TheCPU.fpus &= ~(FPUS_C0 | FPUS_C2 | FPUS_C3);
 			goto fp_notok;
 		   default: /* FNENI,FNDISI: 8087 */
 			    /* FSETPM,FRSTPM: 80287 */
-			goto fp_ok;	// do nothing
+			break;	// do nothing
 		   }
 		   break;
 
@@ -1180,7 +1180,6 @@ fp_notok:
 	// should be caught by Fp87_illegal_op
 	dosemu_error("Unknown FPop %x.%d\n", exop, reg);
 	}
-fp_ok:
 }
 
 int Fp87_illegal_op(int exop, int reg)

